app/views/layouts/application.html.haml

```diff
@@ -71,6 +71,12 @@
       / For non-visual user agents:
       #top
         %a.hidden.doNotPrint{ :href => "#primary" } Skip to main content.
+      %div{:class => 'portal-pages-umbrella'}
+        %div{:class => 'portal-pages-umbrella-contain cols'}
+          %div{:class => 'portal-pages-concord-link col-12'}
+            %a{:class => 'portal-pages-concord-link__item', :href => 'https://concord.org/'}
+              Learn about the Concord Consortium
+              %i{:class => 'icon-arrow-diagonal'}
```

Comment:
The current plan is to have this umbrella header only rendered by the PortalPages header that is going to be on every page. And this PortalPages header will be smart about knowing which portals it should render it and which portals it should not.